site/public/js/configuration.js

```diff
@@ -3,13 +3,24 @@ $(document).ready(function() {
     updateEmailSeatingOption();
 
     function updateForumMessage() {
+        updateForumCategoryWarning();
         if ($("#forum-enabled").is(":checked")) {
             $("#forum-enabled-message").show();
         } else {
             $("#forum-enabled-message").hide();
         }
     }
 
+    function updateForumCategoryWarning() {
+        if ($("#forum-category-warning").length) {
```

Comment:
You don't need to perform a check if the element exists, if JQuery can't find the element show/hide/toggle will have no effect

Reply:
Fixed!